charts/csi-baremetal-deployment/templates/configmap/wbt-config.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```

Comment:
Can we have common config map and avoid creating new ones for each parameter?

Reply:
ConfigMap renamed